File: Abstraction/Ad-hoc/area.java
class Main {
    static void printArea(int side) {
        System.out.println("Area of square is: " + side * side);
    }

    static void printArea(int height, int width) {
        System.out.println("Area of rectangle is: " + height * width);
    }

    public static void main(String[] args) {
        printArea(5);
        printArea(4, 6);
    }
}

File: Abstraction/Ad-hoc/area.cs
using System;

class Program {
    static void PrintArea(int side) {
        Console.WriteLine("Area of square is: " + side * side);
    }

    static void PrintArea(int height, int width) {
        Console.WriteLine("Area of rectangle is: " + height * width);
    }

    static void Main() {
        PrintArea(5);
        PrintArea(4, 6);
    }
}
